.circleci/config.yml

```diff
@@ -180,6 +180,33 @@ jobs:
       - *store_test_package_artifacts
       - *store_package_artifacts
 
+  uat_tests:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: UAT Tests
+          command: |
+            epmd -daemon
+            make python-env && make swagger && make multi-build
+            make python-uats
```

Comment:
Are [ws-related python tests](https://github.com/aeternity/epoch/blob/84a9bef744e329f764e1b68966cbc0dfde523512/Makefile#L144) meant to be run too? Deleted?
Cc @velzevur